api/src/certification/shared/domain/models/Version.js

```diff
@@ -13,14 +13,20 @@ export class Version {
     scope: Joi.string()
       .required()
       .valid(...Object.values(Frameworks)),
-    challengesConfiguration: Joi.object().required(),
+    challengesConfiguration: Joi.object()
+      .keys({
+        defaultCandidateCapacity: Joi.number().required(),
+      })
+      .unknown(true)
+      .required(),
   });
 
   /**
    * @param {Object} params
    * @param {number} params.id - version identifier
    * @param {Frameworks} params.scope - Framework scope (CORE, DROIT, etc.)
    * @param {Object} params.challengesConfiguration - Challenges configuration
+   * @param {number} params.challengesConfiguration.defaultCandidateCapacity - capacity when none has been yet determined
```

Comment:
Une fois le Joi changé 

```suggestion
  * @param {FlashAssessmentAlgorithmConfiguration} params.challengesConfiguration
```